### PR TITLE
Feature/timeline update

### PR DESCRIPTION
**Pull request by cadw**
Added full support for the timeline, but still contains a bug when adding a cue. I will need more time to fix this, so we can merge this right now for the presentaition tomorrow.

### CODE REVIEW

File: src/main/webapp/app/timeline/timeline.controller.js [hunk 1]
```diff
@@ -5,97 +5,200 @@
...
+                    cue.camera = newCamera;
+                    cue.cameraAction = newCameraAction;
+                    cue.script = newScript;
+                    cue.timePoint = timePoint;
+
+                    console.log(player, camera, cameraAction, script, timePoint);
+
+                    // Add the Cue to the database
+                    Cue.save(cue);
+                    $state.reload();
```
Comment (rubenwiersma) [inline src/main/webapp/app/timeline/timeline.controller.js]: This is quite a long method. Maybe it can be cut up in different methods.

**Comment by rubenwiersma** [comment]
Not much to remark. Just one large method. We'll merge for functionality, but it'd be good to add an issue

File: src/main/webapp/app/timeline/timeline.controller.js [hunk 2]
```diff
@@ -5,97 +5,200 @@
...
+                });
```
Comment (hungnguyen94) [inline src/main/webapp/app/timeline/timeline.controller.js]: This controller is a bit too long. Maybe put some of the methods in a separate service?

File: src/main/webapp/app/timeline/timeline.controller.js [hunk 3]
```diff
@@ -5,97 +5,200 @@
...
+        function loadPlayers() {
+            Player.query({
 
-        function loadCues () {
-            Cue.query({
+            }, onSuccess, onError);
+
+            function onSuccess(data, headers) {
+                vm.players = data;
+                vm.queryCount = vm.totalItems;
```
Comment (hungnguyen94) [inline src/main/webapp/app/timeline/timeline.controller.js]: Do you actually use queryCount somewhere?

File: src/main/webapp/app/timeline/timeline.controller.js [hunk 4]
```diff
@@ -5,97 +5,200 @@
...
-         */
-        function resizeToParent(canvas) {
-            // Resize it to the parents' size
-            canvas.style.width = '100%';
-            canvas.style.height = '100%';
-
-            // Now set the internal size, so it will not appear blurry
-            canvas.width = canvas.offsetWidth;
-            canvas.height = canvas.offsetHeight;
+        function parseIntAsYear(year) {
```
Comment (hungnguyen94) [inline src/main/webapp/app/timeline/timeline.controller.js]: What's the purpose of this function?

File: src/main/webapp/app/timeline/timeline.controller.js (hunk 2 above)
Comment (rubenwiersma) [inline src/main/webapp/app/timeline/timeline.controller.js]: I've made an issue for this.

**Pull request merged**